### ISSUE TITLE
No API generated when used with `Fastlane.swift`

### ISSUE THREAD

**Issue by BorysShcherbyna** [AckeeCZ/fastlane-plugin-firebase_management#7]
As per [documentation](https://docs.fastlane.tools/getting-started/ios/fastlane-swift/#using-plugins) after adding a plugin Fastlane will automatically generate the corresponding API and make it available in fastlane/swift/Plugins.swift.
However, for fastlane-plugin-firebase_management no API's are generated.

**Comment by janmisar** [comment]
I don't think this is a problem of this specific plugin 🤔 Does it work well for other plugins?

**Comment by BorysShcherbyna** [comment]
I’ve tried [fastlane-plugin-file_manager](https://github.com/jasonnam/fastlane-plugin-file_manager) and apis were generated correctly.
